v-next/hardhat/src/internal/builtin-plugins/solidity-test/task-action.ts

```diff
@@ -38,14 +39,16 @@ const runSolidityTests: NewTaskActionFunction<TestActionArguments> = async (
 ) => {
   let rootFilePaths: string[];
 
-  if (chainType !== "l1") {
-    console.log(
-      chalk.yellow(
-        `Chain type selection for tests will be implemented soon. Please check our communication channels for updates. For now, please run the task without the --chain-type option.`,
-      ),
+  if (!isSupportedChainType(chainType)) {
+    // NOTE: We could make the error more specific here.
```

Comment:
i think this error works

Reply:
I just copied the error from a similar validation in the node plugin, along with the comment. I'll remove it from both.